hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/contract/s3a/ITestS3AContractSeek.java

```diff
@@ -114,6 +127,7 @@ protected Configuration createConfiguration() {
     S3ATestUtils.disableFilesystemCaching(conf);
     conf.setInt(READAHEAD_RANGE, READAHEAD);
     conf.set(INPUT_FADVISE, seekPolicy);
+    conf.set(SSL_CHANNEL_MODE, sslChannelMode.name());
```

Comment:
add this option to the `removeBucketOverrides` list so that no per-bucket setting will interfere with the test

Reply:
Done